test/controllers/gobierto_data/api/v1/datasets_controller_test.rb

```diff
@@ -315,12 +321,28 @@ def test_index_when_md_custom_field_changes_translations_availability
             parsed_csv = CSV.parse(response_data).map { |row| row.map(&:to_s) }
 
             assert_equal active_datasets_count + 1, parsed_csv.count
-            assert_equal %w(id name slug table_name data_updated_at columns category md-without-translations md-with-translations), parsed_csv.first
+            assert_equal %w(id name slug table_name data_updated_at columns category md-without-translations md-with-translations description), parsed_csv.first
             assert_includes parsed_csv, array_data(dataset)
             refute_includes parsed_csv, array_data(other_site_dataset)
           end
         end
 
+        def test_catalog
```

Comment:
This test is quite incomplete:

1. add datasets to the site, so the XML of the dataset can be tested
2. some datasets should be in draft status to check they are not present
3. check attributes of the XML, there are lots of attributes missing there to be reviewed
4. instead of many asset_includes you might check the whole response